R/epi.R

```diff
@@ -1,40 +1,41 @@
-#' Add mortality rate
+#' Adjust severe rates as a result of treatment coverage, dis-aggregated by hospitalised and community and estimate mortality
```

Comment:
This is maybe a little clunky (not sure if this improves it!)
```suggestion
#' Adjust severe rates as a result of treatment coverage and estimate mortality disaggregated by hospitalised and community
```